modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,44 @@
 = What’s New in Studio
+
+Anypoint Studio 7.10 introduces a notification mechanism to help you keep up to date with your modules versions in Exchange, and also enables you to propagate metadata to flows and sub-flows.
+
+== Modules Updates Notifications
+
+Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available.
+
+You can also prompt Studio to check for modules updates:
+
+. Ensure that you are logged in to Anypoint Platform.
+. Select the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button.
++
+image::manage-mule-modules-location.png[]
+
+[CAUTION]
+This feature does not support modules that are inherited from a parent pom file.
+
+See xref:update-modules.adoc[Update Your Modules] for more information.
+
+
+== Automatic Refactor When Renaming a Flow
+
+When you rename a flow in Anypoint Studio, your change is now automatically refactored across your project.
+
+== Metadata Propagation Between Flows
+
+Studio can infer the metadata of your current flow so that, when you use a Flow Reference component to reference another flow, you can set the metadata from your current flow as custom metadata for your referenced flow. You can also extract an operation from your current flow into a new flow or sub-flow and Studio can automatically declare the metadata as custom metadata of a new flow.
```

Comment:
Are these things automatic or does the audience have to configure them? I am guessing the former:
```suggestion
Studio infers the metadata of your current flow so that when you use a Flow Reference component to reference another flow, you can set the metadata from your current flow as custom metadata for your referenced flow. You can also extract an operation from your current flow into a new flow or subflow, and Studio automatically declares the metadata as custom metadata of a new flow.
```